scripts/run_system_check.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+import argparse
+import getpass
+import logging
+import os
+import time
+
+from eventkit_cloud.utils.client import EventKitClient
+
+logger = logging.getLogger(__name__)
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('url', help='The EventKit instance base url (i.e. http://cloud.eventkit.test).')
+    parser.add_argument('--verify', default='',
+                        help='True to enable ssl verification, false to disable ssl verification')
+    parser.add_argument('--certificate', default='',
+                        help='The path to a certificate to use for authentication')
+    parser.add_argument('--full', default='',
+                        help='The path to a certificate to use for authentication')
+
+    args = parser.parse_args()
+    user = password = None
+    certificate = args.certificate
+    if not certificate:
+        user = os.getenv('EVENTKIT_USER')
+        if not user:
+            user = input("EventKit Username: ")
+        password = os.getenv('EVENTKIT_PASS')
+        if not password:
+            password = getpass.getpass("EventKit Password: ")
+
+    if args.verify:
+        verify = False if args.verify.lower() in ['false', 'f'] else args.verify
```

Comment:
For verify to be true, does it need to be in ['t', 'true']. What will happen if it's something else like 'yes' or 'word'. This might not be important in the grand scheme of things.

Could something like this be used: https://stackoverflow.com/a/43357954/2767100

Reply:
I used something like that.  The problem is that for verify it can be True, False, or the path to the certificate file.  A string value may be appropriate.  But you are right that it is easy to intuit "no" for a user.

Reply:
Ah, wondered why it was allowed to be string. Makes sense.